packages/sources/dar/test/integration/adapter.test.ts

```diff
@@ -77,4 +77,9 @@ describe('Price Endpoint', () => {
     const response = await testAdapter.request({ base: 'ETH' })
     expect(response.statusCode).toEqual(400)
   }, 30000)
+
+  it('should return error (unsubscribed asset pair)', async () => {
```

Comment:
I think this test name is slightly misleading - BTC hasn't been unsubscribed, instead we're returning an error because our mock fixtures don't include BTC data.